mock-connector/src/test/scala/it/agilelab/darwin/connector/mock/MockConnectorSpec.scala

```diff
@@ -8,14 +11,77 @@ import org.scalatest.matchers.should.Matchers
 
 class MockConnectorSpec extends AnyFlatSpec with Matchers {
 
-  it should "load the test schemas" in {
-    new MockConnectorCreator().create(ConfigFactory.empty()).fullLoad() should have size (2)
-  }
-
-  it should "load the test schemas and custom ones" in {
+  it should "load the schema manually inserted" in {
     val connector = new MockConnectorCreator().create(ConfigFactory.empty())
     connector.insert((3L, Schema.create(Type.BYTES)) :: Nil)
-    connector.fullLoad() should have size (3)
+    connector.fullLoad() should have size 1
+  }
+
+  it should "load the schema automatically from resources" in {
+    val connector = new MockConnectorCreator().create(ConfigFactory.parseMap {
+      new java.util.HashMap[String, Object] {
+        put(ConfigurationKeys.RESOURCES, util.Arrays.asList("test/MockClassAlone.avsc", "test/MockClassParent.avsc"))
+      }
+    })
+    connector.fullLoad() should have size 2
+  }
+
+  it should "load the schema automatically from files" in {
+    val p = Paths.get(".")
+      .resolve("mock-connector")
+      .resolve("src")
+      .resolve("test")
+      .resolve("resources")
+      .resolve("test")
```

Comment:
p could be a class-scoped variable since all the test resources for this suite are kept there